ingress/watch-resource/Dockerfile

```diff
@@ -0,0 +1,21 @@
+# Copyright 2016 The Kubernetes Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM alpine:3.4
+
+COPY watch-resource-cmdrunner /
```

Comment:
Would be nice to make this part of the generic https example also: https://github.com/kubernetes/kubernetes/issues/30532, since it is a modular sidecar that watches and reloads